kong/kong.lua

```diff
@@ -308,6 +308,7 @@ function Kong.balancer()
   current_try.port = addr.port
 
   -- set the targets as resolved
+  ngx.log(ngx.DEBUG, "setting address (try ", addr.try_count, "): ",addr.ip,":",addr.port)
```

Comment:
perf: on this hot code path, let's cache `ngx.log` and `ngx.DEBUG` which imply global accesses.

style: it seems like we are going over the 80 chars limit and missing a few spaces for readability. Let's use the following form instead:
```lua
  ngx.log(ngx.DEBUG, "setting address (try ", addr.try_count, "): ",
                     addr.ip, ":", addr.port)
```

(with globals caching ofc)